### PR TITLE
Make the docker container tag configurable.

### PR DESCRIPTION
**Pull request by lance**
Do not merge this yet. It still needs documentation. @lholmquist since this is my first contribution, maybe you want to take a look to make sure it's done in a way that makes sense for you.

### CODE REVIEW

**Comment by lance** [comment]
Connects to: #38

**Comment by lholmquist** [comment]
👀

File: lib/definitions/build-config-spec.js [hunk 1]
```diff
@@ -13,8 +13,9 @@ function createBuildConfigSpec (options) {
   options = options || {};
   const buildConfigSpec = baseBuildConfigSpec;
   buildConfigSpec.source = buildSource('Binary');
-  buildConfigSpec.strategy = buildStrategy();
+  buildConfigSpec.strategy = buildStrategy(options);
   buildConfigSpec.output = buildOutput(options.outputImageStreamTag);
+  console.log('BuildConfigSpec', buildConfigSpec);
```
Comment (lholmquist) [inline lib/definitions/build-config-spec.js]: probably don't need this

Comment (lance) [inline lib/definitions/build-config-spec.js]: Oops! debugging..

**Pull request merged**